enterprise/frontend/src/metabase-enterprise/remote_sync/components/ChangesLists/CollectionPath.tsx

```diff
@@ -0,0 +1,38 @@
+import { Fragment } from "react";
+
+import { collection as collectionUrl } from "metabase/lib/urls";
+import { Anchor, Group, Text } from "metabase/ui";
+
+import type { CollectionPathSegment } from "../../utils";
+
+interface CollectionPathProps {
+  segments: CollectionPathSegment[];
+}
+
+export const CollectionPath = ({ segments }: CollectionPathProps) => {
```

Comment:
Did we try using one of the existing `CollectionBreadcrumb` components? If not, it might be worth checking out rather than making another one